Refit.Tests/RestService.cs

```diff
@@ -419,5 +425,92 @@ public void NonRefitInterfacesThrowMeaningfulExceptions()
             // We should get an InvalidOperationException if we call a method without a base address set
             await Assert.ThrowsAsync<InvalidOperationException>(async () => await fixture.GetUser(null));
         }
+
+        [Fact]
+        public async Task SimpleDynamicQueryparametersTest()
+        {
+
+            var myParams = new MySimpleQueryParams();
+            myParams.FirstName = "John";
+            myParams.LastName = "Rambo";
+
+            var fixture = RestService.For<IHttpBinApi<HttpBinGet, MySimpleQueryParams, int>>("https://httpbin.org/get");
+
+            var resp = await fixture.Get(myParams, 99);
```

Comment:
can we stub this test out? Don't use a real service as it may cause intermittent test failures. There are other examples in the tests of stubbing out a service.

Reply:
Are you referring to using a MockHttpMessageHandler?